cmd/juju/commands/ssh_unix_test.go

```diff
@@ -241,41 +242,43 @@ func (s *SSHSuite) TestSSHWillWorkInUpgrade(c *gc.C) {
 	}
 }
 
-func (s *SSHSuite) TestSSHCommandHostAddressRetryAPIv1(c *gc.C) {
-	s.setHostDialerFunc(func(address string) error {
-		return network.NoAddressError("public")
-	})
-	s.setForceAPIv1(true)
-
-	s.testSSHCommandHostAddressRetry(c, false)
-}
-
-func (s *SSHSuite) TestSSHCommandHostAddressRetryAPIv2(c *gc.C) {
-	s.setHostDialerFunc(func(address string) error {
-		return network.NoAddressError("available")
-	})
-	s.setForceAPIv1(false)
-
-	s.testSSHCommandHostAddressRetry(c, false)
-}
-
-func (s *SSHSuite) TestSSHCommandHostAddressRetryProxyAPIv1(c *gc.C) {
-	s.setHostDialerFunc(func(address string) error {
-		return network.NoAddressError("private")
-	})
-	s.setForceAPIv1(true)
-
-	s.testSSHCommandHostAddressRetry(c, true)
-}
-
-func (s *SSHSuite) TestSSHCommandHostAddressRetryProxyAPIv2(c *gc.C) {
-	s.setHostDialerFunc(func(address string) error {
-		return network.NoAddressError("available")
-	})
-	s.setForceAPIv1(false)
-
-	s.testSSHCommandHostAddressRetry(c, true)
-}
+/// XXX(jam): 2017-01-25 do we need these functions anymore? We don't really
```

Comment:
I realized when looking again, only half of them are testing v1.  I can mimic what they were doing by changing it to force failing all ssh addresses, which gets them to pass instead. (which I've done)